File: javascript/6_kyu/twoSum.js
// Two Sum
// https://www.codewars.com/kata/52c31f8e6605bcc646000082/train/javascript

function twoSum(n, t) {
    for (var i = 0; i < n.length; i++) {
        if ( !(i == n.length) ) {
            for (var j = i + 1; j < n.length; j++) {
                if (n[i] + n[j] == t) {
                    return [i, j];
                }
            }
        }
    }
}

console.log(twoSum([1,2,3], 4));
console.log(twoSum([1234,5678,9012], 14690));
console.log(twoSum([2,2,3], 4));

File: python/6_kyu/two_sum.py
# Two Sum
# https://www.codewars.com/kata/52c31f8e6605bcc646000082/train/python

def two_sum(n, t):
    for i in range(len(n)):
        if i != len(n):
            for j in range(i + 1, len(n)):
                if n[i] + n[j] == t:
                    return (i, j)
    
print(two_sum([1,2,3], 4))
print(two_sum([1234,5678,9012], 14690))
print(two_sum([2,2,3], 4))